tools/install-vs-addons.cmd

```diff
@@ -0,0 +1,39 @@
+set "PATH=C:\Windows;C:\Windows\System32;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files\Git\bin"
+cd %~dp0
+call powershell -File .\install_llvm-win64.ps1
+
+REM Download Visual Studio LLVM extension required for clang build to succeed
+call download.cmd https://llvmextensions.gallerycdn.vsassets.io/extensions/llvmextensions/llvm-toolchain/1.0.363769/1560930595399/llvm.vsix
+
+REM Install optional components required for ARM build - vs2017-BuildTools
+IF EXIST "%ProgramFiles(x86)%\Microsoft Visual Studio\2017\BuildTools" (
+"%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vs_installer.exe" ^
+	-- modify --installPath "%ProgramFiles(x86)%\Microsoft Visual Studio\2017\BuildTools" -q ^
```

Comment:
Curious whether the `-- ` before `modify` is necessary?

Reply:
Good question. Let me follow-up on this.

Reply:
I think it wasn't necessary. Not sure where it came from. It was benign and I removed the empty param.